data/index.js

```diff
@@ -64,7 +64,7 @@ function addStudentsToCourse(studentIds, courseId) {
 			}
 
 			return students().then((collection) => {
-				return collection.update({_id: {$in: studentIds}}, {$addToSet: {courses: courseInfo}});
+				return collection.update({_id: {$in: studentIds}}, {$addToSet: {courses: courseInfo}}, {multi: true});
```

Comment:
Allows mongo to update multiple entries. Otherwise it will just update the first matched entry. Found this out while debugging when I tried to add a course that had 2 students in it instead of just 1.